docs/integrations/sources/postgres.md

```diff
@@ -94,7 +94,7 @@ These are the additional steps required (after following the [quick start](#quic
 
 We recommend following the steps in the [quick start](#quick-start) section to confirm that Airbyte can connect to your Postgres database prior to configuring CDC settings.
 
-For CDC, you must connect to primary/master databases. Pointing the connector configuration to replica database hosts for CDC will lead to failures.
+For CDC from stand-by read-replicas, the required minimum version of Postgres is 16.
```

Comment:
Yes, because logical decoding on standbys is possible [only since PG 16](https://www.postgresql.org/docs/release/16.0/#:~:text=Allow%20logical%20decoding%20on%20standbys%20(Bertrand%20Drouvot%2C%20Andres%20Freund%2C%20Amit%20Khandekar))

Reply:
So the version gate is not for the query, it is for attempting to replicate from a read-replica. In this case for older versions we would run into different errors unrelated to this change. 
The query is perfectly executable for older versions of PG (from 9.5 onwards) since it uses 
`pg_last_wal_receive_lsn()` and `pg_is_in_recovery()` functions that are both available in past version as well as the already used `pg_current_wal_lsn()`.

Reply:
To summarise. Airbyte can replicate from PG starting from 9.5 according to the docs. But to replicate from a read-replica, one needs PG 16 because it is only possible to perform cascading replications in general (regardless from airbyte or not) [only from PG 16 onwards ](https://github.com/airbytehq/airbyte/pull/45397/#:~:text=standbys%20is%20possible-,only%20since%20PG%2016,-Contributor)

Reply:
Thanks for the clarification!